### ISSUE TITLE
After disabling the extension, orders can no longer be opening in Magento

### ISSUE THREAD

**Issue by phillipmeyer** [PHOENIX-MEDIA/Magento-WorldPay#17]
Hello,

We have used this extension for many years but have recently moved our payment provider away from WorldPay. However, after disabling the extension (by renaming Phoenix_Worldpay.xml) we find that we cannot access any of our orders that were paid using WorldPay. Is there a workaround for this so that we don't have to keep the extension live indefinitely to be able to access old orders?

Here's the error from the exception log:
exception 'Mage_Core_Exception' with message 'The requested Payment Method is not available.' in /var/www/html/app/Mage.php:595

**Comment by phoenix-bjoern** [comment]
Disalbe the module on website level and leave it enabled globally.

**Comment by phillipmeyer** [comment]
Hi,

Thanks for the prompt feedback but ideally I want to minimise the number of extensions that we have installed so I wanted to completely remove the extension. If I remove the extension then I can't access the orders that were paid using the extension. Magento just crashes.

Is there any way around this?

I can leave the extension installed but disable WorldPay as a payment option from the Configuration-PaymentMethods page but I would prefer to completely remove the extension as we do not envisage using WorldPay again.

**Comment by phoenix-bjoern** [comment]
Nope. Magento saves the module with the order and tries to load it. The generated exception show you the details.
So the described way is already best practice. Or you change the column with the used payment module to smth else but then the payment information can't be rendered correctly in the backend as the wrong template file is used.
